source/runtime.cpp

```diff
@@ -875,7 +885,7 @@ void reshade::runtime::save_config(const std::filesystem::path &path) const
 
 void reshade::runtime::load_preset(const std::filesystem::path &path)
 {
-	const ini_file preset(path);
+	const ini_file preset(g_reshade_dll_path.parent_path() / path);
```

Comment:
I'd rather force `_current_preset_path` to always be absolute rather than doing this here.